src/vos/vos_overhead.c

```diff
@@ -11,6 +11,12 @@ vos_pool_get_msize(void)
 	return sizeof(struct vos_pool_df);
 }
 
+int
+vos_pool_get_scm_cutoff(void)
```

Comment:
is this really necessary? can't use VOS_BLK_SZ in tests?

Reply:
I suppose we could